scripts/zones/AlTaieu/mobs/Ulxzomit.lua

```diff
@@ -3,75 +3,83 @@
 --  Mob: Ul'xzomit
 -----------------------------------
 require("scripts/globals/follow")
+require("scripts/globals/status")
 -----------------------------------
 local entity = {}
 
-entity.onMobSpawn = function(mob)
+function entity.onMobSpawn(mob)
```

Comment:
We follow the deleted method. For consistency ><

Reply:
No prob. It was a change I overlooked when porting my fix from my own server.

Reply:
Definitely want to avoid.